install/setup.sql

```diff
@@ -918,7 +922,7 @@ INSERT IGNORE INTO `{PREFIX}system_settings`
 ('unauthorized_page','3'),
 ('site_status','1'),
 ('auto_template_logic','{AUTOTEMPLATELOGIC}'),
-('default_template','3'),
```

Comment:
Das wird schief gehen, da die Ids bestimmt willkürlich vergeben werden. Ich würde diesen Parameter einfach löschen oder auf 0 setzen und mit dem Installer vom RMS das Template setzen. Bei einer Leerinstallation würde ich die Wahl einfach dem Nutzer überlassen. Auch beim RMS finde ich die Wahl von JEasyUI als Standard-Template aufgrund der Lizenz-Probleme mehr als fraglich. Ich würde mich mit AdminLTE wohler fühlen. Fur die Standard-Login-Seite habe ich z.B.gar kein template gesetzt und alles in die Seite selber integriert.

Reply:
P. S. mit dem RMS-Installer kannst du über Template-Namen nach der richtigen ID suchen - da sind die templates ja schon installiert.

Reply:
Das Problem mit dem default-Template ist folgendes: Wenn ich eine Seite im root hinzufüge bekommt diese das default-Template (an anderen Stellen bekommt sie das template vom parent vererbt). Hat das default-Template jetzt nicht die erforderlichen TVs dann werden UID und die anderen TV-Werte für diese Seite nicht gespeichert.
Das default-Template muss also noch vor der Installation von Exface passend gesetzt werden. Also entweder wir weisen dem Modx-Default-Template die TVs zu, oder das default Template wird während der Modx-Installation entsprechend gesetzt (momentaner Stand, könnte man auch noch auf AdminLTE ändern) oder es wird nach der Modx-Installation aber vor der Exface-Installation manuell gesetzt.
Da der obige Code ('default_template', '5') nur bei der Installation ausgeführt wird, wird er meiner Meinung nach solange gültig sein bis bei der Installation weitere Templates hinzugefügt werden, die die Ids u.U. verändern. Dann müsste auch das default-Template angepasst werden.

Reply:
Habe ich entfernt.